Multisig/UI/App/UIKit/CollectiblesViewController/CollectiblesViewController.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  CollectiblesViewController.swift
+//  Multisig
+//
+//  Created by Andrey Scherbovich on 02.11.20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import UIKit
+
+class CollectiblesViewController: LoadableViewController, UITableViewDelegate, UITableViewDataSource {
+    let clientGatewayService = App.shared.clientGatewayService
```

Comment:
I think, this should be a variable and not a constant so that this dependency is potentially changea-ble

Reply:
ok